src/chainparams.cpp

```diff
@@ -331,6 +331,11 @@ class SigNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 1199145601; // January 1, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = 1230767999; // December 31, 2008
 
+        // Deployment of Taproot (BIPs 340-342)
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1199145601; // January 1, 2008
```

Comment:
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = Consensus::BIP9Deployment::ALWAYS_ACTIVE;
```

Should be always active

Reply:
I think the ideal behaviour might be to have it always active on the default global signet, but optional for custom signets. Without the ability to disable it for custom signets, it's not possible to make a custom signet that behaves the same as mainnet.

Something like https://github.com/ajtowns/bitcoin/commit/4ceae373743043fac372a27244538a2ee64d0295 perhaps

Reply:
Well, testnet/regtest already have a way to configure these in the config file... maybe just copying/sharing that logic?

Reply:
`-vbparams` is regtest-only, and I think more general than we'd want for something on signet. (I don't think signalling activations on signet is ever actually useful, in particular)

Reply:
Made it always active for now. Adding a new option for custom signets goes beyond a bugfix.